portfolio/src/main/java/com/google/sps/servlets/CommentsListServlet.java

```diff
@@ -15,31 +16,57 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import java.time.format.DateTimeFormatter;
+import java.time.LocalDateTime; 
 
-/** Servlet responsible for listing tasks. */
+
+/** Servlet responsible for listing comments. */
 @WebServlet("/comments-list")
 public class CommentsListServlet extends HttpServlet {
-
+  
+  private DatastoreService datastore; 
+  
+  @Override
+  public void init(){
+    datastore = DatastoreServiceFactory.getDatastoreService();
+  } 
+  
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    int numComments = getMax(request); 
```

Comment:
This can actually be replaced with (what I consider to be) a clearer one-liner:


```java
    int maxComments = Integer.parseInt(request.getParameter("max")); 
```